packages/syft/src/syft/service/code/user_code.py

```diff
@@ -729,6 +739,119 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
         else:
             raise NotImplementedError
 
+    def _ephemeral_node_call(
+        self, time_alive=None, n_consumers=None, *args: Any, **kwargs: Any
+    ) -> Any:
+        # relative
+        from ... import _orchestra
+
+        # Right now we only create a number of workers
+        # In the future we might need to have the same pools/images as well
+
+        if n_consumers is None:
+            print(
+                SyftInfo(
+                    message="Creating a node with n_consumers=2 (the default value)"
+                )
+            )
+            n_consumers = 2
+
+        if time_alive is None and "blocking" in kwargs and not kwargs["blocking"]:
+            print(
+                SyftInfo(
+                    message="Closing the node after time_alive=300 (the default value)"
+                )
+            )
+            time_alive = 300
+
+        # This could be changed given the work on containers
+        ep_node = _orchestra().launch(
+            name=f"ephemeral_node_{self.func_name}_{random.randint(a=0, b=10000)}",  # nosec
+            reset=True,
+            create_producer=True,
+            n_consumers=n_consumers,
```

Comment:
we should set the deploy_to argument here to `python`, otherwise it can also spin up docker containers as deployment_type is also derived from environment variables.